woocommerce/woocommerce
# Dev: remove vscode typescript build task

- dev: remove vscode tsc build task
- dev: remove .gitignore entry for vscode build task

### All Submissions:






### Changes proposed in this Pull Request:

This VScode build task was previously added to allow Typescript checking without enforcing it during build, as enforcement at that point in time was not viable due to numerous Typescript violations.

The build process enforcement has since been enabled and this task is not required anymore. If you rely on this, you may still continue to use it by creating a local copy of the task definition. But otherwise you may continue to catch Typescript errors by running the build process, for example:

`pnpm run start` while in `./plugins/woocommerce-admin/`


### How to test the changes in this Pull Request:

1. Project should still build fine and any TS errors should be caught by the build process

### Other information:




### FOR PR REVIEWER ONLY:
